Frontend/ui-common/src/Components/PopUpModal/Controller.purs

```diff
@@ -680,6 +729,54 @@ config = {
         , padding : (Padding 0 0 0 0)
       }
     }
+  , deliveryDetailsConfig : dummyDeliveryDetailsConfig
 }
 
+dummyDeliveryDetailsConfig :: DeliveryDetailsConfig
+dummyDeliveryDetailsConfig = 
+  let config' = dummyDeliveryPrimaryText
+  in
+    {
+      visibility : GONE,
+      margin : Margin 0 0 0 0,
+      isSource : true,
+      locationTitle : "",
+      locationDetails : "",
+      personNameDetails : config',
+      mobileNumberDetails : config',
+      addressDetails : config',
+      instructionDetails : config',
+      checkBoxDetails : {
+        text : ""
+        , isSelected : false
+        , visibility : true
+      }
+    }
 
+dummyDeliveryPrimaryText :: PrimaryEditTextController.Config
+dummyDeliveryPrimaryText = 
+  let
+      config = PrimaryEditTextController.config
+      primaryEditTextConfig' = config
+        { editText
+          { color = Color.black800
+          , singleLine = true
+          , placeholder = "9999......"
+          , textStyle = FontStyle.SubHeading3
+          }
+        , background = Color.white900
+        , topLabel
+          { text = "Enter Number"
```

Comment:
don't give any dummy text here. Keep it empty. 

Send the translated text from Screen instead.